src/TuDa.CIMS.Web/Services/IWorkingGroupApi.cs

```diff
@@ -51,7 +52,7 @@ public async Task<ErrorOr<IEnumerable<WorkingGroup>>> GetAllAsync() =>
     /// </returns>
     /// <remarks>This will be generated by <see cref="Refit"/>.</remarks>
     [Get("/")]
-    protected Task<IApiResponse<IEnumerable<WorkingGroup>>> GetAllAsyncInternal();
+    protected Task<IApiResponse<IEnumerable<WorkingGroup>>> GetAllAsyncInternal([FromQuery] string? name);
```

Comment:
🔧 You need to use the RefitAttribute here
```suggestion
    protected Task<IApiResponse<IEnumerable<WorkingGroup>>> GetAllAsyncInternal([Query] string? name);
```